File: Python/Python - EASY/PART - 005/Count primes in a range.py
m = int(input())
n = int(input())

count = 0

def is_prime(n):
    if n < 2:
        return False
    for i in range(2, int(n**.5) + 1):
        if n % i == 0:
            return False
    return True

for i in range(m, n+1):
    if is_prime(i):
        count += 1
print(count)

File: C++/Easy/PART - 005/Count primes in a range.cpp
#include <iostream>
#include <cmath>

bool is_prime(int n) {
    if (n < 2) return false;
    for (int i = 2; i <= std::sqrt(n); ++i) {
        if (n % i == 0) return false;
    }
    return true;
}

int main() {
    int m, n, count = 0;
    std::cin >> m >> n;

    for (int i = m; i <= n; ++i) {
        if (is_prime(i)) {
            count++;
        }
    }

    std::cout << count << std::endl;

    return 0;
}
